parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -509,10 +509,11 @@ configGPUDrivers() {
   retrycmd_if_failure 120 5 25 mount -t overlay -o lowerdir=/usr/lib/x86_64-linux-gnu,upperdir=${GPU_DEST}/lib64,workdir=${GPU_DEST}/overlay-workdir none /usr/lib/x86_64-linux-gnu || exit {{GetCSEErrorCode "ERR_GPU_DRIVERS_CONFIG"}}
   retrycmd_if_failure 3 1 600 sh $GPU_DEST/nvidia-drivers-$GPU_DV --silent --accept-license --no-drm --dkms --utility-prefix="${GPU_DEST}" --opengl-prefix="${GPU_DEST}" || exit {{GetCSEErrorCode "ERR_GPU_DRIVERS_START_FAIL"}}
   echo "${GPU_DEST}/lib64" >/etc/ld.so.conf.d/nvidia.conf
+  cp "${GPU_DEST}/bin/*" /usr/bin
```

Comment:
This seems to be creating a second copy of the file, should we create a symlink at `/usr/bin/nvidia-smi` to `/usr/local/nvidia/bin/nvidia-smi` instead?

Reply:
Actually we can ` mv "${GPU_DEST}/bin/*" /usr/bin ` if you has this concern, since utility libraries have been added to system library path with `echo "${GPU_DEST}/lib64" >/etc/ld.so.conf.d/nvidia.conf`.

It will work with a symlink, but there're also other utility bins and compute bins in `${GPU_DEST}/bin/*`, and I would like to keep the change small to not introduce any potential issue.

Reply:
mv works for me, thanks!